packages/database/test/helpers/util.ts

```diff
@@ -21,10 +21,23 @@ import '../../index';
 import { Reference } from '../../src/api/Reference';
 import { Query } from '../../src/api/Query';
 import { ConnectionTarget } from '../../src/api/test_access';
-import { RepoInfo } from '../../src/core/RepoInfo';
 
 export const TEST_PROJECT = require('../../../../config/project.json');
 
+const USE_EMULATOR = process.env.USE_RTDB_EMULATOR;
+const EMULATOR_PORT = process.env.RTDB_EMULATOR_PORT;
+const EMULATOR_NAMESPACE = process.env.RTDB_EMULATOR_NAMESPACE;
+
+export const DATABASE_ADDRESS = USE_EMULATOR
+  ? `http://localhost:${EMULATOR_PORT}`
+  : TEST_PROJECT.databaseURL;
+
+export const DATABASE_URL = USE_EMULATOR
+  ? `http://localhost:${EMULATOR_PORT}?ns=${EMULATOR_NAMESPACE}`
+  : TEST_PROJECT.databaseURL;
```

Comment:
I stared at these a bit to figure out what the difference between DATABASE_ADDRESS and DATABASE_URL is.  It would be slightly easier if we defined them in relation to each other (and added a comment).  E.g.:

```
// When running against the emulator, the hostname will be "localhost" rather than 
// something like  "namespace.firebaseio.com" and so we need to append the namespace 
// as a query param.
export const DATABASE_URL = USE_EMULATOR ? DATABASE_ADDRESS + '?ns=${EMULATOR_NAMESPACE}` : DATABASE_ADDRESS;
```

Reply:
Done.